pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1643,6 +1643,14 @@ func TestReconcilePodUpdateStatus(t *testing.T) {
 		t.Fatalf("Did not get expected condition %s", diff.PrintWantGot(d))
 	}
 
+	label, ok := newTr.ObjectMeta.Labels[taskNameLabelKey]
+	if !ok {
+		t.Errorf("Labels were not added to task run")
+	}
+	if ld := cmp.Diff(label, "test-task"); ld != "" {
```

Comment:
Thanks for this! Couple very small nits:
- `PrintWantGot` expects args to have been passed to `cmp.Diff` in the order `(expected, actual)` but I think they're reversed here?  I'm actually not 100% sure this matters with plain string comparisons - we could use a `if label != "test-task"` here I think instead?  But it's better to keep the `cmp.Diff` ordering consistent in case this code gets copy-pasted by future test writers.
- Suggest putting "test-task" into a named variable so it can be shared between line 1606 and here, showing the connection between those two values.

Reply:
Thanks for the review. Updates made.